website/docs/ckeditor-source.md

```diff
@@ -56,19 +56,19 @@ test("single paragraph", () => {
   let doc = CKEditorSource.fromRaw(ckDoc).canonical();
 
   expect(doc.content).toBe("Here is a paragraph");
-  expect(doc.canonical().annotations.sort(compareAnnotations)).toMatchObject([
+  expect(doc.canonical().annotations.sort()).toMatchObject([
```

Comment:
```suggestion
  expect(doc.canonical().sort()).toMatchObject([
```

Reply:
Actually `doc` is already canonicalized, we don't need to sort at all.